### PR TITLE
[hi] Localize app-container.md

### PR DESCRIPTION
**Pull request by OtherHorizon**
Signed-off-by: Shaurya Ayush [](mailto:shauryachoudhary2015@gmail.com)

Localizes `content/en/docs/reference/glossary/app-container.md` to Hindi `content/hi/docs/reference/glossary/app-container.md`

Mentioned Issue:- https://github.com/kubernetes/website/issues/35600

### CODE REVIEW

**Comment by taniaduggal** [comment]
@OtherHorizon @bishal7679 This file is already localized by me, please @OtherHorizon checked once which files are not localized, then work on that.

File: content/hi/docs/reference/glossary/app-container.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: ऐप कंटेनर (App Container)
+id: app-container
+date: 2019-02-12
+full_link:
+short_description: >
+  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। इनिट कंटेनर के साथ तुलना करें।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/app-container.md:7]:
```suggestion
  एक कंटेनर कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। इनिट कंटेनर के साथ तुलना करें।
```

**Review by Babapool — COMMENTED** [review]
A small nit from my side, otherwise LGTM.

**Comment by divya-mohan0209** [comment]
@OtherHorizon : Would you be able to amend the page per feedback from @Babapool & squash your commits? We'd be happy to ready this for merge.

**Comment by Babapool** [comment]
@anubha-v-ardhan since this PR has only a minor nit (non-blocking as well imo), could we merge this PR and then create a follow up PR/issue to fix this? Thanks

**Review by anubha-v-ardhan — COMMENTED** [review]
/lgtm
/approve

Thank you @OtherHorizon

**Pull request merged**